src/client/Microsoft.Identity.Client/ApiConfig/SystemWebViewOptions.cs

```diff
@@ -1,169 +1,176 @@
+        /// <remarks>Takes precedence over <see cref="HtmlMessageError"/></remarks>
+        public Uri BrowserRedirectError { get; set; }
+
+        /// <summary>
+        /// This hides privacy prompt displayed on the iOS Devices (ver 13.0+) when set to true.
+        /// By default, it is false and displays the prompt.
+        /// </summary>
+        public bool HidePrivacyPrompt { get; set; } = false;
+
+        /// <summary>
+        /// Allows developers to implement their own logic for starting a browser and navigating to a specific Uri. MSAL
+        /// will use this when opening the browser. Leave it null and the user configured browser will be used.
+        /// Consider using the static helpers OpenWithEdgeBrowserAsync and OpenWithChromeEdgeBrowserAsync
+        /// </summary>
+        public Func<Uri, Task> OpenBrowserAsync { get; set; }
+
+        internal void LogParameters(ICoreLogger logger)
+        {
+            logger.Info("DefaultBrowserOptions configured");
+
+            logger.InfoPii("HtmlMessageSuccess " + HtmlMessageSuccess,
+                "HtmlMessageSuccess? " + !String.IsNullOrEmpty(HtmlMessageSuccess));
+            logger.InfoPii("HtmlMessageError " + HtmlMessageError,
+               "HtmlMessageError? " + !String.IsNullOrEmpty(HtmlMessageError));
+            logger.InfoPii("BrowserRedirectSuccess " + BrowserRedirectSuccess,
+               "BrowserRedirectSuccess? " + (BrowserRedirectSuccess != null));
+            logger.InfoPii("BrowserRedirectError " + BrowserRedirectError,
+               "BrowserRedirectError? " + (BrowserRedirectError != null));
+            logger.Info($"HidePrivacyPrompt {HidePrivacyPrompt}");
+        }
+
+        internal static void ValidatePlatformAvailability()
+        {
+#if !SUPPORTS_OS_SYSTEM_BROWSER
+            throw new PlatformNotSupportedException(
+                "WithSystemWebViewOptions API is not supported on this platform. " +
+                "No options are available on other platforms.");
+#endif
+        }
+
+#if SUPPORTS_OS_SYSTEM_BROWSER
+
+        /// <summary>
+        /// Use Microsoft Edge to navigate to the given URI. On non-windows platforms it uses 
+        /// whatever browser is the default.
+        /// </summary>
+        public static async Task OpenWithEdgeBrowserAsync(Uri uri)
+        {
+            if (uri == null)
+            {
+                throw new ArgumentNullException(nameof(uri));
+            }
+
+            string url = uri.AbsoluteUri;
+
+#if DESKTOP
+            url = url.Replace("&", "^&");
+            Process.Start(new ProcessStartInfo("cmd", $"/c start microsoft-edge:{url}") { CreateNoWindow = true });
+            await Task.FromResult(0).ConfigureAwait(false);
+#else
+
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                url = url.Replace("&", "^&");
+                Process.Start(new ProcessStartInfo("cmd", $"/c start microsoft-edge:{url}") { CreateNoWindow = true });

```

Comment:
Becuase you included this in the Android and iOS, it causes a crash. Process is not available - see the PR build failure ... `src\client\Microsoft.Identity.Client\ApiConfig\SystemWebViewOptions.cs(122,17): Error CS0103: The name 'Process' does not exist in the current context`

Reply:
Updated the flags